packages/accordion/index.tsx

```diff
@@ -53,12 +53,13 @@ function ResizeHandle({
   return (
     <div
       className={classNames(
-        "absolute w-full border-chrome hover:border-blue-400",
+        "absolute w-full hover:border-blue-400",
```

Comment:
Any Tailwind border color style I tried declaring– even built-in colors– got dropped. I don't know why this is. It does not seem worth spending much time debugging right now though.